source/scripts/index.js

```diff
@@ -0,0 +1,42 @@
+require('jquery');
+require('jquery-ui-bundle');
+
+class RangeSlider {
+    constructor(selector) {
+        this.selector = selector;
+        this.sliderSelector = $(selector);
+        this.min = this.sliderSelector.data("min");
+        this.max = this.sliderSelector.data("max");
+        this.mindefault = this.sliderSelector.data("mindefault");
+        this.maxdefault = this.sliderSelector.data("maxdefault");
+        this.step = this.sliderSelector.data("step");
```

Comment:
i może lepiej zrobić `this.options` i tam umieścić te wszystkie `min, max, step`